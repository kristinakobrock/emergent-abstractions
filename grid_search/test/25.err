  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:02, 22.84it/s] 14%|#############                                                                                | 7/50 [00:00<00:01, 27.76it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 28.42it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 28.39it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 25.54it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 22.68it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 24.35it/s] 50%|##############################################                                              | 25/50 [00:01<00:01, 23.91it/s] 56%|###################################################5                                        | 28/50 [00:01<00:01, 21.50it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 23.32it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 22.96it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 22.83it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 23.54it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 24.98it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 24.99it/s] 98%|##########################################################################################1 | 49/50 [00:02<00:00, 25.18it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.33it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.86it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 22.05it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 27.05it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.20it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.12it/s]
