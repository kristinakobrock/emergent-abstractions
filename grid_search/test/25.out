# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6640515327453613, "acc": 0.6281993389129639, "length": 3.506770610809326, "mode": "train", "epoch": 1}
{"loss": 0.6717002391815186, "acc": 0.620312511920929, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6586088538169861, "acc": 0.6319878697395325, "length": 3.452255964279175, "mode": "train", "epoch": 2}
{"loss": 0.6637149453163147, "acc": 0.6277710199356079, "length": 2.79296875, "mode": "test", "epoch": 2}
{"loss": 0.6463146805763245, "acc": 0.6359261870384216, "length": 3.4999282360076904, "mode": "train", "epoch": 3}
{"loss": 0.6398025751113892, "acc": 0.6585143804550171, "length": 3.9765625, "mode": "test", "epoch": 3}
{"loss": 0.6099417209625244, "acc": 0.6782827973365784, "length": 3.7778069972991943, "mode": "train", "epoch": 4}
{"loss": 0.6628276705741882, "acc": 0.6449218392372131, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.5935505628585815, "acc": 0.6918549537658691, "length": 3.834590196609497, "mode": "train", "epoch": 5}
{"loss": 0.6772019267082214, "acc": 0.6367188096046448, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.5855996608734131, "acc": 0.7018794417381287, "length": 3.841951608657837, "mode": "train", "epoch": 6}
{"loss": 0.6884912848472595, "acc": 0.6597656607627869, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.5793049335479736, "acc": 0.7077037692070007, "length": 3.8631832599639893, "mode": "train", "epoch": 7}
{"loss": 0.6594436168670654, "acc": 0.6572265028953552, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.57062828540802, "acc": 0.7141589522361755, "length": 3.893503189086914, "mode": "train", "epoch": 8}
{"loss": 0.6486836671829224, "acc": 0.6798829436302185, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.5491734743118286, "acc": 0.7275756597518921, "length": 3.9151973724365234, "mode": "train", "epoch": 9}
{"loss": 0.6171259880065918, "acc": 0.7089843153953552, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.5281459093093872, "acc": 0.7466568350791931, "length": 3.946633815765381, "mode": "train", "epoch": 10}
{"loss": 0.6153793931007385, "acc": 0.7314453125, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.5113341212272644, "acc": 0.762858510017395, "length": 3.9644038677215576, "mode": "train", "epoch": 11}
{"loss": 0.6050082445144653, "acc": 0.7339844107627869, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.4978083372116089, "acc": 0.7675780653953552, "length": 3.9751386642456055, "mode": "train", "epoch": 12}
{"loss": 0.620198130607605, "acc": 0.7261719107627869, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.491136759519577, "acc": 0.7700170874595642, "length": 3.982405185699463, "mode": "train", "epoch": 13}
{"loss": 0.5950735807418823, "acc": 0.7330078482627869, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.48527660965919495, "acc": 0.770077109336853, "length": 3.983875274658203, "mode": "train", "epoch": 14}
{"loss": 0.5935134291648865, "acc": 0.7371092438697815, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.4821923077106476, "acc": 0.7718192934989929, "length": 3.9870026111602783, "mode": "train", "epoch": 15}
{"loss": 0.5985039472579956, "acc": 0.7333984971046448, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.4794355034828186, "acc": 0.7707567811012268, "length": 3.9883272647857666, "mode": "train", "epoch": 16}
{"loss": 0.602357029914856, "acc": 0.7322264909744263, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.47343865036964417, "acc": 0.7732774019241333, "length": 3.9882988929748535, "mode": "train", "epoch": 17}
{"loss": 0.5866458415985107, "acc": 0.7365235686302185, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.4690718948841095, "acc": 0.7751914262771606, "length": 3.990295886993408, "mode": "train", "epoch": 18}
{"loss": 0.585243284702301, "acc": 0.7312500476837158, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.46729201078414917, "acc": 0.7778381109237671, "length": 3.9913182258605957, "mode": "train", "epoch": 19}
{"loss": 0.5723746418952942, "acc": 0.7294921875, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.4573826491832733, "acc": 0.77992182970047, "length": 3.992422103881836, "mode": "train", "epoch": 20}
{"loss": 0.5771816968917847, "acc": 0.7310547232627869, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.45661965012550354, "acc": 0.784869909286499, "length": 3.9924545288085938, "mode": "train", "epoch": 21}
{"loss": 0.5610977411270142, "acc": 0.7347655892372131, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.4498084485530853, "acc": 0.7825109958648682, "length": 3.9934396743774414, "mode": "train", "epoch": 22}
{"loss": 0.5517673492431641, "acc": 0.7390624284744263, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.4470956325531006, "acc": 0.7880042195320129, "length": 3.994037628173828, "mode": "train", "epoch": 23}
{"loss": 0.5288758277893066, "acc": 0.7353516221046448, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.4419402480125427, "acc": 0.7903196215629578, "length": 3.995619058609009, "mode": "train", "epoch": 24}
{"loss": 0.5213658809661865, "acc": 0.7416015267372131, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4364132285118103, "acc": 0.7966342568397522, "length": 3.996025323867798, "mode": "train", "epoch": 25}
{"loss": 0.5138208270072937, "acc": 0.7552734017372131, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.43340402841567993, "acc": 0.7938105463981628, "length": 3.996274709701538, "mode": "train", "epoch": 26}
{"loss": 0.5243634581565857, "acc": 0.7525390982627869, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.43163150548934937, "acc": 0.7969842553138733, "length": 3.9958293437957764, "mode": "train", "epoch": 27}
{"loss": 0.5181898474693298, "acc": 0.7468749284744263, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.42580878734588623, "acc": 0.8017132878303528, "length": 3.996572256088257, "mode": "train", "epoch": 28}
{"loss": 0.5116512179374695, "acc": 0.7501953840255737, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.42657312750816345, "acc": 0.7996373176574707, "length": 3.9954962730407715, "mode": "train", "epoch": 29}
{"loss": 0.5113915801048279, "acc": 0.7500000596046448, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.41738781332969666, "acc": 0.8032615780830383, "length": 3.9970004558563232, "mode": "train", "epoch": 30}
{"loss": 0.5059568285942078, "acc": 0.7585937976837158, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.41393885016441345, "acc": 0.8063286542892456, "length": 3.9963409900665283, "mode": "train", "epoch": 31}
{"loss": 0.5038794279098511, "acc": 0.7525390982627869, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.4081748425960541, "acc": 0.8112900853157043, "length": 3.9968788623809814, "mode": "train", "epoch": 32}
{"loss": 0.4797067940235138, "acc": 0.7710937857627869, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.39605310559272766, "acc": 0.8143147826194763, "length": 3.9974217414855957, "mode": "train", "epoch": 33}
{"loss": 0.48760470747947693, "acc": 0.7662109732627869, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.3886777460575104, "acc": 0.8200920820236206, "length": 3.9968934059143066, "mode": "train", "epoch": 34}
{"loss": 0.4795316159725189, "acc": 0.7718749642372131, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.38342422246932983, "acc": 0.8210473656654358, "length": 3.998326539993286, "mode": "train", "epoch": 35}
{"loss": 0.47212502360343933, "acc": 0.7775390148162842, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.37653690576553345, "acc": 0.8231697678565979, "length": 3.997850179672241, "mode": "train", "epoch": 36}
{"loss": 0.4807127118110657, "acc": 0.7755860090255737, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.3757767379283905, "acc": 0.8284299373626709, "length": 3.998427629470825, "mode": "train", "epoch": 37}
{"loss": 0.4829360842704773, "acc": 0.7820312976837158, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.36569616198539734, "acc": 0.8330835700035095, "length": 3.998112201690674, "mode": "train", "epoch": 38}
{"loss": 0.47708961367607117, "acc": 0.7763671875, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3622128665447235, "acc": 0.8356320261955261, "length": 3.9984962940216064, "mode": "train", "epoch": 39}
{"loss": 0.4735797941684723, "acc": 0.7845702767372131, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.3605130612850189, "acc": 0.8398832678794861, "length": 3.9987051486968994, "mode": "train", "epoch": 40}
{"loss": 0.4771876335144043, "acc": 0.78515625, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.3553924560546875, "acc": 0.8406520485877991, "length": 3.998940944671631, "mode": "train", "epoch": 41}
{"loss": 0.4781390130519867, "acc": 0.7785156965255737, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.35214096307754517, "acc": 0.8428918123245239, "length": 3.998633623123169, "mode": "train", "epoch": 42}
{"loss": 0.46205928921699524, "acc": 0.7833985090255737, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.34904155135154724, "acc": 0.8432535529136658, "length": 3.9989852905273438, "mode": "train", "epoch": 43}
{"loss": 0.4822152256965637, "acc": 0.783203125, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.34706228971481323, "acc": 0.8451442122459412, "length": 3.9990456104278564, "mode": "train", "epoch": 44}
{"loss": 0.47549962997436523, "acc": 0.7830077409744263, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.34383851289749146, "acc": 0.8478564620018005, "length": 3.9988832473754883, "mode": "train", "epoch": 45}
{"loss": 0.45005637407302856, "acc": 0.7953124642372131, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.34285488724708557, "acc": 0.8487462997436523, "length": 3.9990508556365967, "mode": "train", "epoch": 46}
{"loss": 0.46218162775039673, "acc": 0.7882813215255737, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.33966347575187683, "acc": 0.8479692339897156, "length": 3.998321294784546, "mode": "train", "epoch": 47}
{"loss": 0.4540433883666992, "acc": 0.791015625, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.3350127637386322, "acc": 0.8501672148704529, "length": 3.9993643760681152, "mode": "train", "epoch": 48}
{"loss": 0.45016583800315857, "acc": 0.7982421517372131, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.3309790790081024, "acc": 0.8567789793014526, "length": 3.999493360519409, "mode": "train", "epoch": 49}
{"loss": 0.4456480145454407, "acc": 0.79296875, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.3307648301124573, "acc": 0.8534159064292908, "length": 3.99955415725708, "mode": "train", "epoch": 50}
{"loss": 0.4508967697620392, "acc": 0.7966795563697815, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.3253861963748932, "acc": 0.8574889898300171, "length": 3.999560832977295, "mode": "train", "epoch": 51}
{"loss": 0.45958492159843445, "acc": 0.7853515148162842, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.32977744936943054, "acc": 0.8545353412628174, "length": 3.9993996620178223, "mode": "train", "epoch": 52}
{"loss": 0.4638710916042328, "acc": 0.79296875, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.32281577587127686, "acc": 0.8578442931175232, "length": 3.999650001525879, "mode": "train", "epoch": 53}
{"loss": 0.455697238445282, "acc": 0.7947266101837158, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.3202165365219116, "acc": 0.8596146702766418, "length": 3.9996047019958496, "mode": "train", "epoch": 54}
{"loss": 0.4501914083957672, "acc": 0.7986327409744263, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.313921183347702, "acc": 0.8625644445419312, "length": 3.9996540546417236, "mode": "train", "epoch": 55}
{"loss": 0.46112972497940063, "acc": 0.7947264313697815, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.31087034940719604, "acc": 0.8640439510345459, "length": 3.9996912479400635, "mode": "train", "epoch": 56}
{"loss": 0.45769989490509033, "acc": 0.7919921875, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.3109457492828369, "acc": 0.86634761095047, "length": 3.999809503555298, "mode": "train", "epoch": 57}
{"loss": 0.46896958351135254, "acc": 0.7986327409744263, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.3110879361629486, "acc": 0.8639864325523376, "length": 3.999805212020874, "mode": "train", "epoch": 58}
{"loss": 0.45441871881484985, "acc": 0.8048828840255737, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3069755434989929, "acc": 0.8652274012565613, "length": 3.999866485595703, "mode": "train", "epoch": 59}
{"loss": 0.45930030941963196, "acc": 0.8005858659744263, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.3047660291194916, "acc": 0.8704096674919128, "length": 3.999530792236328, "mode": "train", "epoch": 60}
{"loss": 0.4519326984882355, "acc": 0.8023437857627869, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.74564528465271
